index.html

```diff
@@ -1,60 +1,217 @@
-                        </p>
-                    </div>
+  <div class="container">
+    <div class="main">
+      <header>
+        <div class="main-logo"><a class="logo" href="./index.html"> John Doe</a>
+        </div>
+        <nav>
+          <ul class="menu-bar">
+            <li class="menu-items">Portfolio</li>
+            <li class="menu-items">About</li>
+            <li class="menu-items">Contact</li>
+            <li><img src="./png/nav-icon.png" alt="Multi-Post Stories"></li>
+          </ul>
+          <div class="hamburger" onclick="displayContent()">
+            <img src="./svg/menu.svg" alt="Menu" />
+          </div>
+        </nav>
+      </header>
+      <section class="main-section" id="portfolio">
+        <div class="background-adjust">
+          <div class="details">
+            <h1>Hey There. I`m Esmatullah</h1>
+            <h2>I`am a Software Developer</h2>
+            <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don`t hestiate to contact me.
+            </p>
+          </div>
+          
+          <ul class="social-media">
+            <li><a href="#"><img src="./svg/github.svg" alt="github"/></a></li>
+            <li><a href="#"><img src="./svg/linkedin.svg" alt="linkedin"/></a></li>
+            <li><a href="#"><img src="./svg/Vector3.svg" alt="angelist"/></a></li>
+            <li><a href="#"><img src="./svg/twitter.svg" alt="twitter"/></a></li>
+            <li><a href="#"><img src="./svg/Vector5.svg" alt="medium"/></a></li>
+          </ul>
+        </div>
+        <div class="background-desktop">
+        </div>
+      </section>
+ 
+  </div>
+  <section class="work">
+    <a href="#work"></a>
 
-                    <ul class="social-media">
-                        <li><a href="#"><img src="./svg/github.svg" alt="github" /></a></li>
-                        <li><a href="#"><img src="./svg/linkedin.svg" alt="linkedin" /></a></li>
-                        <li><a href="#"><img src="./svg/Vector3.svg" alt="angelist" /></a></li>
-                        <li><a href="#"><img src="./svg/twitter.svg" alt="twitter" /></a></li>
-                        <li><a href="#"><img src="./svg/Vector5.svg" alt="medium" /></a></li>
-                    </ul>
-                </div>
-                <div class="background-desktop">
-                </div>
-            </section>
+    <div class="work-1">
+      <div class="work1-visual">
+        <h1>My Recent Works</h1>
+        <div class="line"></div>
+      </div>
+      <div class="work1-content">
+        <div class="work-placeholder"><img src="./svg/ImgPlaceholder.svg" alt="" />
         </div>
+        <div class="work-data">
+          <div class="content-data">
+            <h1>Multi-Post Stories</h1>
```

Comment:
- [ ] [REQUIRED] Please, make sure to use a `h2` or `h3` instead of `h1` for card title because there should be only one `h1` in the whole page for semantic document